lib/Swagger/src/TemplateClassification/Storage/Api/TCSApi.php

```diff
@@ -129,31 +137,27 @@ public function getTemplateDetailsWithHttpInfo($wiki_id, $page_id)
         if ($page_id === null) {
             throw new \InvalidArgumentException('Missing the required parameter $page_id when calling getTemplateDetails');
         }
-  
         // parse inputs
         $resourcePath = "/{wiki_id}/{page_id}/providers";
         $httpBody = '';
-        $queryParams = array();
-        $headerParams = array();
-        $formParams = array();
-        $_header_accept = ApiClient::selectHeaderAccept(array('application/json'));
+        $queryParams = [];
+        $headerParams = [];
+        $formParams = [];
+        $_header_accept = $this->apiClient->selectHeaderAccept(['application/json']);
         if (!is_null($_header_accept)) {
             $headerParams['Accept'] = $_header_accept;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Accept" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)